src/onboarding/admin.js

```diff
@@ -0,0 +1,61 @@
+function add2CommunityGroup(user) {
+  var resultSheet = SpreadsheetApp.getActiveSpreadsheet().getSheetByName("Emails");
+  var userRowIndex = searchRow(user, resultSheet);
+  var newUserRowRange = resultSheet.getRange(userRowIndex, 5); // get the "Groups added" cell
```

Comment:
should not rely on absolute column numbers. 

I have a function getHeaderMapping to enable lookup of header row -> header column number:

https://github.com/coderbunker/gas/blob/master/src/timesheet/data.js

turn that into a reusable library and have it in both